tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -287,14 +284,50 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 		oa.StopInsertDataTo(&clusterA)
 	})
 
-	ginkgo.It("Adhoc backup and restore with BR CRD", func() {
-		if framework.TestContext.Provider != "aws" {
-			framework.Skipf("provider is not aws, skipping")
+	ginkgo.It("BackupAndRestoreWithBR", func() {
```

Comment:
```suggestion
	ginkgo.It("Backup and restore with BR", func() {
```

describe in sentence form